### PR TITLE
Stage removal

### PR DESCRIPTION
**Pull request by NissimHadar**
# Desciption
Removes the Stage from the Zone Propeties UI and the C++ code behind.
Includes a protocol change.
The only effect of the change is the removal of the Stage section in Zone Properties.

# Testing
## General
After installing the PR, verify that the local domain appears as before (to verify usage with older content).  Check that the modes in the zone parameters are as expected (keylight, skybox, ambient and haze).
Check that the inherit modes are correct (the ambient light is quite dark, but not black like in 'Off')

## UI tests
This verifies that the Zone properties page looks and runs as expected.
Follow the procedure in:  https://highfidelity.testrail.net/index.php?/cases/view/1815&group_by=cases:title&group_order=asc&group_id=-1

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6163/app_12128_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6163/HighFidelity-Beta-PR12128-46b494c80231ad6900a64261be4c242a02887240.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6163/HighFidelity-Beta-PR12128-46b494c80231ad6900a64261be4c242a02887240.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6179/app_12128_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6179/HighFidelity-Beta-PR12128-ea10bfba6cafc36f5fa2b05af5cf8185187fa5ba.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6179/HighFidelity-Beta-PR12128-ea10bfba6cafc36f5fa2b05af5cf8185187fa5ba.dmg

**Comment by QAadahlin** [comment]
Currently Testing

**Comment by QAadahlin** [comment]
Currently testing

**Comment by QAadahlin** [comment]
@conklin94122 Completed Testing for PR#12128. PASSED.

File: libraries/networking/src/udt/PacketHeaders.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ PacketVersion versionForPacketType(PacketType packetType) {
         case PacketType::EntityEdit:
         case PacketType::EntityData:
         case PacketType::EntityPhysics:
-            return static_cast<PacketVersion>(EntityVersion::ZoneLightInheritModes);
+            return static_cast<PacketVersion>(EntityVersion::StageRemoved);
```
Comment (samcake) [inline libraries/networking/src/udt/PacketHeaders.cpp]: INstead of StageREmoved, could you call that "ZoneSTageRemoved" ?

**Comment by NissimHadar** [comment]
Done.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6192/app_12128_PR.apk

**Comment by conklin94122** [comment]
Code changed, please retest.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6192/HighFidelity-Beta-PR12128-dff47f38a70dca6664d6d9d5b996dc9ae7a09f57.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6192/HighFidelity-Beta-PR12128-dff47f38a70dca6664d6d9d5b996dc9ae7a09f57.dmg

**Comment by QAadahlin** [comment]
Currently testing

**Comment by QAadahlin** [comment]
@conklin94122 Testing completed for PR#12128. PASSED.

**Pull request merged**